test/test_jit.py

```diff
@@ -13158,6 +13155,10 @@ def fn(*inputs, **kwargs):
                             check_against_reference(self, traced_fn,
                                                     fn, (self_variable,) + args_variable, kwargs_variable,
                                                     check_types=check_types)
+                            # Fuser not supported on windows
```

Comment:
what's the reason that fuser not support in windows?

Reply:
idk, it never has been. probably because something isn't supported in the g++ -> dlopen flow on windows